src/quo2/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -34,6 +34,7 @@
                              :i/derivation-path
                              {:color               label-color
                               :size                32
-                              :accessibility-label :derivation-path-label}])]))))
+                              :accessibility-label :derivation-path-label}]
+           nil)]))))
```

Comment:
By this `nil` and `condp` vals, I'd say we could use `case` again, since we are comparing keywords and `case` is simpler to understand than `condp`.

Reply:
Done